add-domain

```diff
@@ -284,6 +285,7 @@ mikatafa.weebly.com
 mivezosikobo.weebly.com
 moxalabukeziro.weebly.com
 munuteme.weebly.com
+netflixxcxx.weebly.com
```

Comment:
Just as info, weebly removes these in less than an hour if you inform them, that is way faster and better approach, than adding them to any blacklists

Reply:
They told me that they were unable to verify that it was phishing...

Reply:
hmm ok... that was a first... but have been told there is a first time for everything.